lib/helpers/lang_helper.dart

```diff
@@ -0,0 +1,192 @@
+  "ht": "kreyòl ayisyen",
+  "hu": "magyar nyelv",
+  "hy": "Հայերէն Hayerèn",
+  "hz": "Otjiherero",
+  "id": "bahasa Indonesia",
+  "ig": "Asụsụ Igbo",
+  "ii": "ꆈꌠꉙ Nuosuhxop",
+  "ik": "Iñupiaq",
+  "is": "íslenska",
+  "it": "italiano",
+  "iu": "ᐃᓄᒃᑎᑐᑦ Inuktitut",
+  "ja": "日本語 Nihongo",
+  "jv": "ꦧꦱꦗꦮ / Basa Jawa",
+  "ka": "ქართული Kharthuli",
+  "ki": "Gĩkũyũ",
+  "kk": "қазақ тілі qazaq tili",
+  "km": "ភាសាខ្មែរ Phiəsaakhmær",
+  "kn": "ಕನ್ನಡ Kannađa",
+  "ko": "한국어 Han'gug'ô",
+  "ks": "कॉशुर / كأشُر",
+  "ku": "kurdî / کوردی",
+  "kv": "Коми кыв",
+  "kw": "Kernowek",
+  "ky": "кыргызча kırgızça",
+  "la": "Lingua latīna",
+  "lb": "Lëtzebuergesch",
+  "lg": "Luganda",
+  "li": "Lèmburgs",
+  "lo": "ພາສາລາວ Phasalaw",
+  "lt": "lietuvių kalba",
+  "lu": "Kiluba",
+  "lv": "Latviešu valoda",
+  "mh": "Kajin M̧ajeļ",
+  "mi": "Te Reo Māori",
+  "mk": "македонски јазик makedonski jazik",
+  "ml": "മലയാളം Malayāļã",
+  "mn": "монгол хэл mongol xel",
+  "mr": "मराठी Marāţhī",
+  "ms": "Bahasa Melayu",
+  "mt": "Malti",
+  "my": "မြန်မာစာ Mrãmācā",
+  "na": "dorerin Naoero",
+  "nb": "norsk bokmål",
+  "nd": "siNdebele saseNyakatho",
+  "ne": "नेपाली भाषा Nepālī bhāśā",
+  "ng": "ndonga",
+  "nl": "Nederlands",
+  "nn": "norsk nynorsk",
+  "no": "norsk",
+  "nr": "isiNdebele seSewula",
+  "nv": "Diné bizaad",
+  "ny": "Chichewa",
+  "oc": "occitan",
+  "om": "Afaan Oromoo",
+  "or": "ଓଡ଼ିଆ",
+  "os": "Ирон æвзаг Iron ævzag",
+  "pa": "ਪੰਜਾਬੀ / پنجابی Pãjābī",
+  "pi": "Pāli",
+  "pl": "Język polski",
+  "ps": "پښتو Pax̌tow",
+  "pt": "português",
+  "qu": "Runa simi",
+  "rm": "Rumantsch",
+  "rn": "Ikirundi",
+  "ro": "limba română",
+  "ru": "русский язык russkiĭ âzık",
+  "rw": "Ikinyarwanda",
+  "sa": "संस्कृतम् Sąskŕtam",
+  "sc": "sardu",
+  "sd": "سنڌي / सिन्धी / ਸਿੰਧੀ",
+  "se": "davvisámegiella",
+  "sg": "yângâ tî sängö",
+  "si": "සිංහල Sĩhala",
+  "sk": "slovenčina",
+  "sl": "slovenski jezik",
+  "sm": "Gagana faʻa Sāmoa",
+  "sn": "chiShona",
+  "so": "af Soomaali",
+  "sq": "Shqip",
+  "sr": "српски / srpski",
+  "ss": "siSwati",
+  "st": "Sesotho [southern]",
+  "su": "Basa Sunda",
+  "sv": "svenska",
+  "sw": "Kiswahili",
+  "ta": "தமிழ் Tamił",
+  "te": "తెలుగు Telugu",
+  "tg": "тоҷикӣ toçikī",
+  "th": "ภาษาไทย Phasathay",
+  "ti": "ትግርኛ",
+  "tk": "Türkmençe / Түркменче / تورکمن تیلی تورکمنچ",
+  "tl": "Wikang Tagalog",
+  "tn": "Setswana",
+  "to": "lea faka-Tonga",
+  "tr": "Türkçe",
+  "ts": "Xitsonga",
+  "tt": "татар теле / tatar tele / تاتار",
+  "ty": "Reo Tahiti; Reo Mā'ohi",
+  "ug": "ئۇيغۇرچە ",
+  "uk": "Українська мова; Українська",
+  "ur": "اُردُو Urduw",
+  "uz": "Oʻzbekcha / Ózbekça / ўзбекча / ئوزبېچه",
+  "ve": "Tshivenḓa",
+  "vi": "Tiếng Việt",
+  "wa": "Walon",
+  "xh": "isiXhosa",
+  "yi": "ייִדיש",
+  "yo": "èdè Yorùbá",
+  "za": "Vahcuengh / 話僮",
+  "zh": "中文 Zhōngwén",
+  "zu": "isiZulu"
+};
+
+// languages not in this list will be marked as BETA
+final List<String> completeLanguages = [
```

Comment:
All Languages synced here?

Reply:
In weblate currently Norwegian Bokmål, Greek and Esperanto are only partially translated.